src/org/labkey/test/ModulePropertyValue.java

```diff
@@ -81,16 +81,32 @@ public ModulePropertyValue(String moduleName, String containerPath, String prope
         _inputType = inputType;
     }
 
+    public ModulePropertyValue(String moduleName, String containerPath, String propertyName, String propertyLabel, Object value, InputType inputType)
+    {
+        super(moduleName, containerPath, propertyName, propertyLabel, value);
+        _inputType = inputType;
+    }
+
     public ModulePropertyValue(String moduleName, String containerPath, String propertyName, String value)
     {
         this(moduleName, containerPath, propertyName, value, InputType.text);
     }
 
+    public ModulePropertyValue(String moduleName, String containerPath, String propertyName, String propertyLabel, String value)
+    {
+        this(moduleName, containerPath, propertyName, propertyLabel, value, InputType.text);
+    }
+
     public ModulePropertyValue(String moduleName, String containerPath, String propertyName, boolean value)
     {
         this(moduleName, containerPath, propertyName, value, InputType.checkbox);
     }
 
+    public ModulePropertyValue(String moduleName, String containerPath, String propertyName, String propertyLabel, boolean value)
+    {
+        this(moduleName, containerPath, propertyName, propertyLabel, value, InputType.checkbox);
+    }
+
```

Comment:
Optional: might be time to consider a builder pattern to replace these six constructors.

Reply:
Would make sense, but I'm going to defer that change to later.